front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -126,6 +123,20 @@ const LinearScaleControl = ({
     event.preventDefault();
   };
 
+  // Put all labels from the UI Schema in an array
+  const labelsFromSchema = Array.from({ length: maximum }).map((_, index) => {
+    if (uischema?.options?.[`linear_scale_label${index + 1}`]) {
+      return uischema.options[`linear_scale_label${index + 1}`];
+    }
+  });
```

Comment:
Minor: Since the second argument of Array.from can be a map function, we can do that directly there

```suggestion
  const labelsFromSchema = Array.from({ length: maximum }, (_, index) =>
    uischema?.options?.[`linear_scale_label${index + 1}`]
  ).filter(Boolean);
```

Reply:
Ah I didn't know you could do this - thanks!! :)